metrics/init.lua

```diff
@@ -56,6 +56,32 @@ end
 local function clear()
     global_metrics_registry.collectors = {}
     global_metrics_registry.callbacks = {}
+    global_metrics_registry.label_pairs = {}
+end
+
+local function set_global_labels(label_pairs)
+    checks('?table')
+
+    label_pairs = label_pairs or {}
+
+    -- Verify label table
+    local string_like = function(obj)
```

Comment:
I know only two cases when tostring can throw an error:
- call without args
- throw an error in __tostring methods

Should we validate errors in such way? I think no... It's quite strange to  validate strings in such way

Reply:
As I said above, I did not find any explicit specification on labels, and implicit ones require only this property. What type restrictions on keys and values I should set?

Reply:
I think `type(label) == 'string'`

Another types I think don't have any sense

Reply:
Changed verification process (with simple test)